cmd/goQuery/cmd/root.go

```diff
@@ -458,7 +458,6 @@ func entrypoint(cmd *cobra.Command, args []string) (err error) {
 			return err
 		}
 		logger.Infof("Status %q: %s", result.Status.Code, result.Status.Message)
-		return nil
```

Comment:
This is the actual "fix" / trick: Just don't abort, both when querying locally or globally we continue to print.